apps/web/src/modules/dao/components/About/DaoDescription.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Button, Flex, Text } from '@zoralabs/zord'
+import HTMLReactParser from 'html-react-parser'
+import React, { useEffect, useMemo, useRef } from 'react'
+import ReactMarkdown from 'react-markdown'
+import rehypeRaw from 'rehype-raw'
+import rehypeSanitize from 'rehype-sanitize'
+import remarkGfm from 'remark-gfm'
+
+import { daoDescription as plainDesciption } from 'src/styles/About.css'
+import { isPossibleMarkdown } from 'src/utils/helpers'
+
+import { UNEXPANDED_BOX_HEIGHT, daoDescription, fadingEffect } from './mdRender.css'
+
+export const DaoDescription = ({ description }: { description?: string }) => {
+  const [isOverHeight, setIsOverHeight] = React.useState(false)
+  const [isExpanded, setIsExpanded] = React.useState(false)
+
+  const textRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (
+      textRef.current &&
+      textRef?.current?.scrollHeight > textRef?.current?.clientHeight
+    ) {
+      console.log('fired')
+      setIsOverHeight(true)
+    }
+  }, [])
+
+  const correctedDescription = React.useMemo(() => {
+    if (typeof description === 'string') {
+      return description.replace(/\\n/g, '\n').replace(/\\r/g, '\r')
```

Comment:
can we add a comment on why this is needed? is this an issue from the markdown editor control?

Reply:
I don't think this was coming from Markdown control as the proposal MD Editor doesn't seem to have this issue, and the code is pretty much identical. Also, when I logged the outputs, I did not see the extra escape.